Documentation/project-docs/benchmarking.md

```diff
@@ -120,17 +120,21 @@ class Program
 
 7. Go to the corresponding CoreCLR source folder where the API you want to change exists (for example `coreclr\src\System.Private.CoreLib\shared\System`)
 8. Apply the optimization that you want to test
-9. Rebuild System.Private.Corelib with your change (optionally adding `-skipnative` if the change is isolated to managed code):
+9. Rebuild System.Private.Corelib with your change (optionally adding `-skipnative -skipbuildpackages` if the change is isolated to managed code):
 
-        C:\Projects\coreclr>build.cmd -release -skiptests -skipnative
+        C:\Projects\coreclr>build.cmd -release -skiptests -skipnative -skipbuildpackages
 
 10. For the next step, you have one of two options:
 
   - Rebuild given CoreFX part in Release:
 
           C:\Projects\corefx>build.cmd -c Release /p:CoreCLROverridePath=C:\Projects\coreclr\bin\Product\Windows_NT.x64.Release
 
-  - OR manually copy over the relevant files from within the root of the coreclr output folder to where `CoreRun.exe` lives within corefx (excluding the subdirectories). This ends up being much faster than the first option and if the only thing that changed is   `System.Private.Corelib.dll`, just copy that over:
+  - Force refresh of CoreCLR hard-link copy:
+   
+          C:\Projects\corefx>msbuild /p:CoreCLROverridePath=C:\Projects\coreclr\bin\Product\Windows_NT.x64.Release\ ./external/runtime/runtime.depproj /p:ConfigurationGroup=Release
```

Comment:
Please prefer build.cmd -restore (with the coreclr override path property) over calling into internals. Restore doesn't take long and is part of the build.cmd api. Also i wouldn't list the manual copy instruction below. Restore should suffice for all use cases.

Reply:
you mean `build.cmd -restore /p:CoreCLROverridePath=C:\git\coreclr\bin\Product\Windows_NT.x64.Release`?

Reply:
Do we should update also https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md#testing-with-private-coreclr-bits?

Reply:
Yes please :)